hadoop-hdds/docs/content/integration/Impala.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Impala
+weight: 4
+menu:
+   main:
+      parent: "Application integrations"
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Starting with version **4.2.0**, Apache Impala provides full support for
+querying data stored in **Apache Ozone**. To utilize this functionality,
+ensure that your Ozone version is **1.4.0** or later.
+
+## Supported Access Protocols
+
+Impala supports the following protocols for accessing Ozone data:
+
+* ofs
+* s3a
+
+Note: The o3fs protocol is **NOT** supported by Impala.
+
+## Supported Bucket Types
```

Comment:
We use bucket types to refer to OBS and FSO. What's described here are replication types.